### PR TITLE
adding pre-commit hook with eslint checking and prettier format

### PR DESCRIPTION
**Pull request by MohammadSommakia**

### CODE REVIEW

**Review by NourSammour — CHANGES_REQUESTED** [review]
your task is related to pre-commit
so why you did changes in react version and bugs urls

**Review by NourSammour — CHANGES_REQUESTED** [review]
fix confilcts

**Comment by MohammadSommakia** [comment]
sorry about that but I did not pay attention

File: package.json [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
+  },
+  "homepage": "https://github.com/NourSammour/biletiniz#readme",
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
     "start": "cross-env NODE_ENV=production node server.js",
     "test": "jest"
   },
-  "dependencies": {
+   "dependencies": {
```
Comment (NourSammour) [inline package.json:16]: please revert this space

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Pull request closed without merging**